### DISCUSSION TITLE
Use other paramters to downgrade or upgrade the severity for a dependabot vulnerability

### DISCUSSION THREAD

**Discussion by allan-cw** [community/community#40629] [Code Security]
### Select Topic Area

Product Feedback

### Body

As vulnerability management evolved, GitHub could consider some qualitative items to determine the actual criticality of a vulnerability
Let's say a repository deals with sensitive data and the repository is also used in a public facing asset.
If GitHub is capable of reading tags, the vulnerability could receive a different score.
this will for sure help prioritizing real critical items.

**Comment by ossanna16** [comment]
Hi there @allan-cw and welcome to our community! Thank you for the feedback!
